theme/templates/pages/menus/dropdown.html

```diff
@@ -1,55 +1,36 @@
-{% load i18n pages_tags mezzanine_tags %}
+{% load mezzanine_tags %}
 {% spaceless %}
-{% if page_branch_in_menu %}
 
-{% if branch_level == 0 %}
 <ul class="nav navbar-nav">
-    {% for page in page_branch %}
-    {% if not has_home and page.is_primary and forloop.first %}
-    <li class="{% if on_home %} active{% endif %}" id="dropdown-menu-home">
-        <a href="{% url "home" %}">{% trans "Home" %}</a>
+    <li {% if request.get_full_path == "/my-resources/" %}class="active"{% endif %} id="dropdown-menu-my-resources">
+        <a href="/my-resources/">My Resources</a>
     </li>
-    {% endif %}
-    {% if page.in_menu %}
-    <li class="{% if page.has_children_in_menu %}dropdown{% endif %}{% if page.is_current_or_ascendant %} active{% endif %}"
-        id="dropdown-menu-{{ page.html_id }}">
-        <a href="{{ page.get_absolute_url }}">
-            {{ page.title }}
-            {% if page.has_children_in_menu %}<span class="caret"></span>{% endif %}
-        </a>
-        {% if page.has_children_in_menu %}{% page_menu page %}{% endif %}
+    <li {% if request.get_full_path == "/search/" %}class="active"{% endif %} id="dropdown-menu-search">
```

Comment:
Using `request.get_full_path` as conditional to set the `active` class is troublesome in this case because URLs to Discover could include variables. For instance, the following url string won't apply the `active` class to the Discover navigation item.
![image](https://user-images.githubusercontent.com/2448568/54034915-54ba8600-4175-11e9-8c36-d6970f267fdb.png)

Reply:
good point.  I think we can just use ```request.path``` then.  I'll give it a try.

Reply:
@sblack-usu Instead of using equal comparison, you could use startswith string operation to set it active if the request path starts with ```/search``` to cover this case, right?

Reply:
request.path is working.